### ISSUE TITLE
ResNet50 Graph 32卡复现

### ISSUE THREAD

**Issue by strint** [Oneflow-Inc/models#104]
下周一前的目标：
	•	ResNet50 单机8卡 完整训练 （AMP 单机8卡，50 Epoch ，bsz_per_gpu 192， 75.9 精度， 2个半小时）
	•	ResNet50 nn.Graph AMP 训练； 加各种优化参数 （欧阳宇）
	•	ResNet50 nn.Graph Consistent 4机32卡 复现 DLPerf 实验

子任务0、验收和review【wenxiao、xiexuan、ouyangyu、depeng】
- 之前resnet50 xiexuan 负责，就由xiexuan指导实现和验收
- 指导实现
- 验收标准
- 参考
  - https://github.com/Oneflow-Inc/OneFlow-Benchmark/blob/4b149db0110add068b87771d9e7c6fe6b6454fbd/Classification/cnns/of_cnn_train_val.py#L44

子任务1、数据并行, ddp fp32 + graph (amp)
- ofrecord reader需要设置 sbp为 s，placement 为单机 8 卡；
- module 的 parameter 需要设置 sbp 为B，placement 为单机8卡，看是否 module 有to_consistent 方法；

子任务2、optimizer
- lr_scheduler = lr warmup + lr decay  cosine scheduler 【xuxiaoyu】
- clip_grad【depeng，需要review】
- sgd 等 zzk pr 合并
- loss scale 两种策略 
- GraphConfig配置项，enable_auto_mixed_precision
- trick：label需要平滑下 lable smooth，参考https://github.com/Oneflow-Inc/OneFlow-Benchmark/blob/master/Classification/cnns/of_cnn_train_val.py#L68
